meterfs/common.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs tests common part
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Tomasz Korniluk
+ *
+ *
+ * %LICENSE%
+ */
+
+#include "common.h"
+
+static struct {
+	char buff[40];
+} common;
+
+int common_preallocOpenFile(const char *name, size_t sectors, size_t filesz, size_t recordsz)
+{
+	int fd;
+
+	TEST_ASSERT_EQUAL(0, file_allocate(name, sectors, filesz, recordsz));
+	snprintf(common.buff, sizeof(common.buff), "/%s", name);
+	fd = file_open(common.buff);
+	TEST_ASSERT_GREATER_OR_EQUAL(0, fd);
+
+	memset(common.buff, 0, sizeof(common.buff));
+
+	return fd;
+}
+
+
+void common_readContent(int fd, size_t offset, void *buff, size_t bufflen, void *content, int contentsz, const char *msg)
+{
+	int i;
+
+	TEST_ASSERT_FALSE_MESSAGE(bufflen < contentsz, "Wrong arguments bufflen < contentsz!");
+
+	memcpy(common.buff, content, contentsz);
+	TEST_ASSERT_EQUAL_MESSAGE(bufflen, file_read(fd, offset, buff, bufflen), msg);
+	for (i = 0; i < bufflen - contentsz; ++i)
+		common.buff[contentsz + i] = 0xff;
```

Comment:
```suggestion
	for (i = contentsz; i < bufflen; ++i)
		common.buff[i] = 0xff;
```
A bit simpler.
Why `contentsz` not as `size_t`?

Reply:
Done.